Module/STIG/Convert/Functions.XccdfXml.ps1

```diff
@@ -237,25 +237,47 @@ function Split-StigXccdf
         $dcStig.Benchmark.id = $msStig.Benchmark.id -replace '_STIG', '_DC_STIG'
         $msStig.Benchmark.id = $msStig.Benchmark.id -replace '_STIG', '_MS_STIG'
 
-        # Remove DC only settings from the MS xml
-        Write-Information -MessageData "Removing Domain Controller settings from Member Server STIG"
+        # Remove DC and Core settings from the MS xml
+        Write-Information -MessageData "Removing Domain Controller and Core settings from Member Server STIG"
         foreach ($group in $msStig.Benchmark.Group)
         {
+            # Remove DC only settings from the MS xml
             if ($group.Rule.check.'check-content' -match "This applies to domain controllers")
             {
                 [void] $msStig.Benchmark.RemoveChild($group)
                 Write-Information -MessageData "Removing $($group.id)"
+                # Continue is used to bypass server core installation check
+                continue
+            }
+
+            # Remove Core only settings from MS XML
+            if ($group.Rule.check.'check-content' -match "For server core installations,")
+            {
+                [void] $msStig.Benchmark.RemoveChild($group)
+                $group.Rule.check.'check-content' = $group.Rule.check.'check-content' -replace "(?=For server core installations,)(?s)(.*$)"
```

Comment:
This line of code is used twice.  Does it need to be listed twice or can we put this at the top of the foreach?

Reply:
I could create a function out of it if necessary but it does need to run in each foreach loop. One for the DC stig and one for the MS stig.

Reply:
I think we are good as is for now unless someone else thinks we need a function.  Maybe we add a function the next time text needs to be modified.